### PR TITLE
fix: make user profile images work again

### PR DESCRIPTION
**Pull request by alexandraB99**

### CODE REVIEW

File: ui/imports/shared/popups/UserStatusContextMenu.qml [hunk 1]
```diff
@@ -70,7 +70,7 @@ PopupMenu {
             hoverEnabled: true
             cursorShape: Qt.PointingHandCursor
             onClicked: {
-                openProfilePopup(profileModel.profile.username, profileModel.profile.pubKey, profileModel.profile.thumbnailImage || "");
+                openProfilePopup(profileModel.profile.username, profileModel.profile.pubKey, profileModule.thumbnailImage || "");
```
Comment (PascalPrecht) [inline ui/imports/shared/popups/UserStatusContextMenu.qml:73]: Some of these have already been addressed in: https://github.com/status-im/status-desktop/pull/3961

@iurimatias fee free to close mine in favour of this

Comment (iurimatias) [reply ui/imports/shared/popups/UserStatusContextMenu.qml:73]: ups... I tested that one first and already merged, seeing this one now

Comment (alexandraB99) [reply ui/imports/shared/popups/UserStatusContextMenu.qml:73]: @PascalPrecht @iurimatias are all issues addressed then? shall this one be closed?

**Review by jrainville — APPROVED** [review]

**Comment by anastasiyaig** [comment]
@alexandraB99 please make it work at this screen as well if possible 

Screenshot 2021-11-05 at 09 58 28

**Comment by alexandraB99** [comment]
> @alexandraB99 please make it work at this screen as well if possible
> 
> Screenshot 2021-11-05 at 09 58 28

@anastasiyaig done

**Comment by anastasiyaig** [comment]
@alexandraB99 thank u!

**Pull request merged**